kie-bom/pom.xml

```diff
@@ -147,12 +147,12 @@
       <dependency>
         <groupId>org.kie.server</groupId>
         <artifactId>kie-server-services-optaplanner</artifactId>
-        <version>${version.org.kie}</version>
+        <version>${version.org.optaplanner}</version>
```

Comment:
in order to make it available you should also add:
```
<version.org.optaplanner>7.73.0.Final</version.org.optaplanner>
```

in `kie-user-bom-parent` [1] which is the actual parent pom of `kie-bom`

[1] https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/kie-user-bom-parent/pom.xml#L52